src/super_gradients/training/datasets/samplers/class_balanced_sampler.py

```diff
@@ -127,8 +127,14 @@ def __init__(
         oversample_aggressiveness: float = 0.5,
         num_samples: Optional[int] = None,
         generator=None,
+        shuffle: bool = True,  # noqa
     ) -> None:
         """
+
+        :param shuffle: This parameter is not used, exists for compatibility purposes
```

Comment:
Maybe itd be wise to give a warning?
I don't remeber whether with the balanced sampler the repeates keep the same order or not but maybe some users would find it confusing.
Lets also set the default accordingly.

Reply:
@shaydeci reverted this change. Seems I can simply do this:
```
train_dataloader_params:
   sampler: 
      ClassBalancedSampler:
          ...
```

And not specify `shuffle` key at all and that does the job.

Reply:
Fixed in ee48f23f75732b8762de489f5fd30460bc480494